.github/workflows/translation-check.yml

```
@@ -2,6 +2,9 @@ name: Verify translations
 
 on:
   workflow_dispatch:
+  pull_request:
+    paths:
+      - 'src/translation/**'
```

This path (and the one below) could theoretically be even more specific as long as the other check is disabled.
PRs which change the script or the workflow itself should probably also run it.

```suggestion
      - 'src/translation/wininstaller/**'
      - 'tools/check-wininstaller-translations.sh'
      - '.github/workflows/translation-check.yml'
```

---

Agree.

---

Could probably also be done for the other path list for `push`?